script.js

```diff
@@ -1,4 +1,129 @@
+// input a choice between scissors paper stone
+// output if the player has won the game against the computer
+// if input is not valid, output will indicate what are the valid choices and remind the player to play them accordingly
+var currentGameMode = "waiting for user name";
+var userName = "";
+var computerChoice = function () {
+  var randomInteger = Math.ceil(Math.random() * 3);
+  var computerPlay = "";
+  switch (randomInteger) {
+    case 1:
+      computerPlay = "scissors";
+      break;
+    case 2:
+      computerPlay = "paper";
+      break;
+    case 3:
+      computerPlay = "stone";
+  }
+  return computerPlay;
+};
+
+var numGames = 0;
+var playerWins = 0;
+var comWins = 0;
+var draws = 0;
+var decision = function (player, computer) {
```

Comment:
great use of function to decide to win or loss for the player, great use of += and break .
 you can try instead of break , you could use return, it should work too